File: python/examples/4.ServerAlwaysUp/run_server.py
from ezpysocket import ezpysocket as ps
import cv2
import numpy as np

if __name__ == "__main__":

    while True:
        # Init
        s = ps.EzPySocket(reconnect_on_address_busy=True)

        print("Receiving image...")
        img = s.receive_image()

        print("Sending image...")
        s.send_image(img)

        s.disconnect()

File: cpp/examples/4.ServerAlwaysUp/run_server.cpp
#include "ezcppsocket.h"

int main(int argc, char const *argv[])
{
	while (true){
		EzCppSocket s = EzCppSocket("127.0.0.1", 10000, 2, 1, false, true, 1, true, 2);

		// Receiving
		std::cout << "Receiving data...\n";
		cv::Mat recv_img = s.readImage();

		// Sending
		cv::Mat img = cv::imread("lena.jpg");
		s.sendImage(img);
	}

	return 0;
}